packages/askui-nodejs/src/lib/interactive_cli/create-example-project.ts

```diff
@@ -103,7 +103,7 @@ export class CreateExampleProject {
       this.helperTemplateConfig['allure_stepreporter_import'] = "import { AskUIAllureStepReporter } from '@askui/askui-reporters';";
       this.helperTemplateConfig['reporter_placeholder'] = 'reporter: new AskUIAllureStepReporter(),';
       this.helperTemplateConfig['allure_stepreporter_attach_video'] = `const video = await aui.readVideoRecording();
-  AskUIAllureStepReporter.attachVideo(video);`;
+  await AskUIAllureStepReporter.attachVideo(video);`;
```

Comment:
Indentation seems off...

Reply:
I thought the same actually 😆 

But when you render the nunchucks template the code is correctly intended only if you write it this way.